### PR TITLE
Update Terminus Name with new process

### PR DESCRIPTION
**Pull request by cal-weng**
- [x] Updated [Manage Accounts with TermiPass ](https://docs.jointerminus.com/how-to/termipass/account/#manage-accounts-with-termipass)with the new Terminus Name creation process
- [x] Updated [Terminus Name intro](https://docs.jointerminus.com/overview/terminus/terminus-name.html)
- [x] Updated related images

### CODE REVIEW

File: docs/overview/terminus/terminus-name.md [hunk 1]
```diff
@@ -25,28 +25,20 @@ By simply replacing the "@" in your Terminus Name with a ".", you get your corre
...
-        * Enter your organization's domain name.
-        * Bind your organization email. Currently, only Gmail and Google Workspace email are supported.
-        * Upon completion, you will receive an Organization Terminus Name.
-3. Backup mnemonic phrase.
-
-    The mnemonic phrase is the ONLY way to recover your DID and Terminus Name. Make sure you [backup your Terminus Name](../../how-to/termipass/account/#backup-mnemonic-phrase)'s mnemonic phrase securely!
+Creating a Terminus Name is your starting point of using Terminus. It involves creating a Terminus Account and binding it to a Terminus Name on the mobile version of TermiPass. Depending on whether you choose to bind a verification credential (VC), you can create a Terminus Name in the following modes:
 
+- [Fast creation without binding VC](../../how-to/termipass/account/index.md#create-terminus-name-fast-without-binding-vc-default-mode) (for individual users).
+- [Advanced creation that requires VC binding](../../how-to/termipass/account/index.md#create-terminus-name-with-vc-advanced-mode) (for both individual and organization users).
```
[the remark this answers is not in the document]
Comment (fnalways) [reply docs/overview/terminus/terminus-name.md:31]: Ensure the anchor link is updated if the source title changes

File: docs/how-to/termipass/account/index.md [hunk 2]
```diff
@@ -4,107 +4,83 @@ outline: [2, 4]
...
-5. Proceed to the **Terminus Name** landing page.
+2. Enter your organization's domain name and click **Continue**. Recheck whether your domain name has been verified and configured if an error occurs.   
 
-If you encounter errors while binding, check if your account is already bound to a **Terminus Name**. If not, please contact us.
+3. Bind your organization email. Currently, only Gmail and Google Workspace email are supported. The binding process is similar with that of [Binding an Individual Terminus Name](#bind-an-individual-terminus-name).
 
-Once at the landing page, wait for [Terminus installation](../../terminus/setup/install/) to complete, then proceed with [activation](../../terminus/setup/wizard.md).
+Upon completion, you will receive an **Organization Terminus Name**.
 
+### Import an existing account
```
Comment (fnalways) [inline docs/how-to/termipass/account/index.md:69]: Should we change it to a level 2 heading?

File: docs/how-to/termipass/account/index.md [hunk 3]
```diff
@@ -4,107 +4,83 @@ outline: [2, 4]
 
 # Manage Accounts with TermiPass
 
-Managing Terminus Accounts is a core function of TermiPass. If you are new to TermiPass and Terminus, you can begin with creating a Terminus Name. This guide will walk you through the process.
+Managing Terminus Accounts is a core function of TermiPass. If you are new to Terminus, you can start by creating a Terminus Name. This guide walks you through the process.
```
Comment (cal-weng) [inline docs/how-to/termipass/account/index.md:7]:
```suggestion
Managing Terminus Accounts is a core function of TermiPass. If you are new to Terminus, you can start by creating a Terminus Name. This guide walks you through the process, and other commonly used account operations as well.
```

File: docs/how-to/termipass/account/index.md (hunk 2 above)
Comment (cal-weng) [inline docs/how-to/termipass/account/index.md:69]:
```suggestion
## Import an existing account
```

File: docs/how-to/termipass/account/index.md [hunk 4]
```diff
@@ -4,107 +4,83 @@ outline: [2, 4]
...
-3. Enter your organization's domain name and click **Continue**. Recheck whether your domain name has been verified and configured if an error occurs.   
-4. Bind your organization email. Currently, only Gmail and Google Workspace email are supported. The binding process is similar with that of an [**Individual Terminus Name**](#individual-terminus-name).
-5. Upon completion, you will receive an **Organization Terminus Name**.
+Incorrect mnemonic phrases will prevent importing.
+Accounts without a bound Terminus Name cannot be imported.
+Accounts with a bound Terminus Name can be imported to any TermiPass.
+:::
 
-### Backup Mnemonic Phrase
+### Backup mnemonic phrase
```
Comment (cal-weng) [inline docs/how-to/termipass/account/index.md:83]:
```suggestion
## Backup mnemonic phrase
```

File: docs/overview/terminus/terminus-name.md [hunk 5]
```diff
@@ -25,28 +25,19 @@ By simply replacing the "@" in your Terminus Name with a ".", you get your corre
...
-        * Join an existing organization or create a new one through Terminus Space.
-        * Enter your organization's domain name.
-        * Bind your organization email. Currently, only Gmail and Google Workspace email are supported.
-        * Upon completion, you will receive an Organization Terminus Name.
-3. Backup mnemonic phrase.
-
-    The mnemonic phrase is the ONLY way to recover your DID and Terminus Name. Make sure you [backup your Terminus Name](../../how-to/termipass/account/#backup-mnemonic-phrase)'s mnemonic phrase securely!
-
+Creating a Terminus Name is your starting point of using Terminus. This process includes setting up your Terminus account and binding it to a Terminus Name. Depending on your preference, you can choose from two creation modes:
+  - [Fast creation (default mode)](../../how-to/termipass/account/index.md#create-terminus-name-fast-without-binding-vc-default-mode): Recommended for individual users who prefer a quick setup without binding a verification credential (VC).
```
Comment (cal-weng) [inline docs/overview/terminus/terminus-name.md:29]:
```suggestion
  - [Fast creation (default mode)](../../how-to/termipass/account/index.md#create-without-vc-binding-default-mode): Recommended for individual users who prefer a quick setup without binding a verification credential (VC). 
```

File: docs/overview/terminus/terminus-name.md [hunk 6]
```diff
@@ -25,28 +25,19 @@ By simply replacing the "@" in your Terminus Name with a ".", you get your corre
...
-        * Enter your organization's domain name.
-        * Bind your organization email. Currently, only Gmail and Google Workspace email are supported.
-        * Upon completion, you will receive an Organization Terminus Name.
-3. Backup mnemonic phrase.
-
-    The mnemonic phrase is the ONLY way to recover your DID and Terminus Name. Make sure you [backup your Terminus Name](../../how-to/termipass/account/#backup-mnemonic-phrase)'s mnemonic phrase securely!
-
+Creating a Terminus Name is your starting point of using Terminus. This process includes setting up your Terminus account and binding it to a Terminus Name. Depending on your preference, you can choose from two creation modes:
+  - [Fast creation (default mode)](../../how-to/termipass/account/index.md#create-terminus-name-fast-without-binding-vc-default-mode): Recommended for individual users who prefer a quick setup without binding a verification credential (VC). 
+  - [Advanced creation](../../how-to/termipass/account/index.md#create-terminus-name-with-vc-advanced-mode): Recommended for individual and organizational users that require enhanced security through VC binding.
```
Comment (cal-weng) [inline docs/overview/terminus/terminus-name.md:30]:
```suggestion
  - [Advanced creation](../../how-to/termipass/account/index.md#create-with-vc-advanced-mode): Recommended for individual and organizational users that require enhanced security through VC binding. 
```

File: docs/how-to/termipass/account/index.md [hunk 7]
```diff
@@ -4,107 +4,81 @@ outline: [2, 4]
...
-:::
-
-### Create an Account
+After you get your Terminus Name, wait for [Terminus installation](../../terminus/setup/install/) to complete, then proceed with [activation](../../terminus/setup/wizard.md).
 
-![alt text](/images/how-to/termipass/new_an_account.png)
+### Create with VC (advanced mode)
 
-Choose to either create a new Terminus Account or import an existing one. 
+To create a unique, unrestricted individual Terminus Name, you need to bind a VC through your preferred social account in advanced mode. Your Terminus Name will be identical to your social account name. To enter this mode:
```
Comment (cal-weng) [inline docs/how-to/termipass/account/index.md:30]: @fnalways

File: docs/overview/terminus/terminus-name.md [hunk 8]
```diff
@@ -25,28 +25,19 @@ By simply replacing the "@" in your Terminus Name with a ".", you get your corre
...
-        * Enter your organization's domain name.
-        * Bind your organization email. Currently, only Gmail and Google Workspace email are supported.
-        * Upon completion, you will receive an Organization Terminus Name.
-3. Backup mnemonic phrase.
-
-    The mnemonic phrase is the ONLY way to recover your DID and Terminus Name. Make sure you [backup your Terminus Name](../../how-to/termipass/account/#backup-mnemonic-phrase)'s mnemonic phrase securely!
-
+Creating a Terminus Name is your starting point of using Terminus. This process includes setting up your Terminus account and binding it to a Terminus Name. Depending on your preference, you can choose from two creation modes:
+  - [Fast creation (default mode)](../../how-to/termipass/account/index.md#create-without-vc-binding-default-mode): Recommended for individual users who prefer a quick setup without binding a verification credential (VC). 
+  - [Advanced creation](../../how-to/termipass/account/index.md#create-with-vc-advanced-mode): Recommended for individual and organizational users that require enhanced security and more distinctive names through VC binding.
```
Comment (cal-weng) [inline docs/overview/terminus/terminus-name.md:30]: @fnalways

**Review by fnalways — APPROVED** [review]

**Pull request merged**